source/cpu/power_stub.go

```diff
@@ -1,3 +1,4 @@
+//go:build !amd64
```

Comment:
what does this has to do with an html change?

Reply:
Nothing 🤨 Thanks for noting, fixed. I don't understand what gofmt magic I managed to do get this...